Resources/Prototypes/_White/Entities/Objects/Weapons/Guns/crossbow.yml

```
@@ -0,0 +1,77 @@
+    containers:
+      ballistic-ammo: !type:Container
+        ents: []
+      cell_slot: !type:ContainerSlot
+      crystal_slot: !type:ContainerSlot
+  - type: PowerCellSlot
+    cellSlotId: cell_slot
+  - type: ItemSlots
+    slots:
+      cell_slot:
+        name: power-cell-slot-component-slot-name-default
+  - type: Stretched 
```

yamllint

[error] trailing spaces

(trailing-spaces)